.idea/sonarlint/issuestore/index.pb

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
kannst du bitte noch ein [.gitignore](https://git-scm.com/docs/gitignore) file erstellen in dem projekt, sodass du nur dateien im projekt commitest, die zu sourcecode gehören und nicht zur konfiguration der IDE? 🤔